### ISSUE TITLE
some problem for training on small size picture

### ISSUE THREAD

**Issue by AihahaFox** [MarvinTeichmann/KittiSeg#63]
Hi, thanks for your work! I tried to train the net with my on dataset, which is about 120*40 size. But the training is not so successful. The accuracy is 1.0 and won't change. I tried to change the learning rate, but do not help.

**Comment by daixiaogang** [comment]
Hi，guys，I also meet this problem 。But my picture is 960*720。

**Comment by MarvinTeichmann** [comment]
> The accuracy is 1.0 and won't change. 

Why is this bad? Accuracy of 1.0 means no training error. What is your validation error? Are you overfitting?

**Comment by daixiaogang** [comment]
I Use the dataset which has more than two classes，but I treat it as two classes(road,background),when I use the dataset to train, the train Accuracy is 1.0. Howerver, when I use the network to segmate , result is very bad.

**Comment by AihahaFox** [comment]
I found the reason. The data itself is not so well, which causes training error. Now, I have no problem， thanks!
